scripts/lib/libvirt/templates/admin-node.xml

```diff
@@ -6,7 +6,7 @@
   <os>
     <type arch='$march' machine='$machine'>hvm</type>
 $osloader
-    <nvram>/var/lib/libvirt/$cloud-admin_VARS.fd</nvram>
+    <nvram>/var/lib/libvirt/cloud-admin_VARS.fd</nvram>
```

Comment:
why the removal of $ here? I think $cloud is the variable in case of multiple clouds running in parallel on the same host.